tests/integration/devfile/cmd_devfile_push_test.go

```diff
@@ -925,6 +925,65 @@ var _ = Describe("odo devfile push command tests", func() {
 		Context("odo push (without -f) is executed", func() {
 			ensureFilesSyncedTest(commonVar.Project, false)
 		})
+
+		When("node-js application is created and pushed", func() {
```

Comment:
Can you move this to the existing `When(creating a nodejs component,{})` block?

Reply:
I think we can move this block. But there are many changes that are introduced when we do `odo create` and we will need to replace it with `devfile-with-MR-CL-CR.yaml` which will remove those changes.
@feloy what do you think?

Reply:
I would prefer to keep it separate, as it is a devfile verion  2.2.0, when the other ones are version 2.0.0. You can indicate the devfile version in this `When` so we can see the difference more easily

Reply:
Make sense, will update accordingly